build-support/bin/BUILD

```diff
@@ -41,15 +41,6 @@ python_binary(
   tags = {'type_checked'},
 )
 
-python_binary(
-  name = 'check_pants_pex_abi',
-  sources = ['check_pants_pex_abi.py'],
```

Comment:
We don't need this anymore because we aren't relying on the Pex to get `native_engine.so`.